### PR TITLE
Added contact details and images

### PR DESCRIPTION
**Pull request by MaggieSchuetz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,8 +39,42 @@ <h2>Selected Publications</h2>
...
+
+<section>
+<h2>I really wanted some clickable icons here, but I'll settle for images for now</h2>
+  <figure>
+  <a href="https://www.linkedin.com/in/dr-magdalena-sch%C3%BCtz-625a781a6">
+    <img src="https://source.unsplash.com/random/600x300"
+    width=70" height="70" alt="">
+ </a>
+</figure> 
+<figcaption>LinkedIn again</figcaption>
```
Comment (mwarnck) [inline index.html:58]: I think the figcaption should be nested in the figure element :) 
You also should check the indentation, I made the same in an earlier commit ;)

File: index.html [hunk 2]
```diff
@@ -38,8 +39,42 @@ <h2>Selected Publications</h2>
     </ul>
   </section>
 
-<section id="contact"></section>
-<h2>Contact</h2>
+<section id="contact">
+  <h2>Contact</h2>
+    <h3>Phone:</h3>
+      <p>01234/56789 </p>
+    <h3>Mail: </h3>
```
Comment (ttaannjjaa) [inline index.html:46]: very good idea i think to use `<h3>` to create new lines 😃

File: index.html (hunk 1 above)
Comment (ttaannjjaa) [inline index.html:58]: I am not sure about this...[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption) says figcaption must be first or last child, i believe, this means wrapped inside `<figure></figure>`

File: index.html [hunk 3]
```diff
@@ -38,8 +39,42 @@ <h2>Selected Publications</h2>
...
+</figure> 
+<figcaption>LinkedIn again</figcaption>
+
+<figure>
+  <a href="https://www.facebook.com">
+    <br /><img src="https://source.unsplash.com/random/600x300"
+    width=70" height="70" alt="">
+ </a>
+</figure> 
+<figcaption>Facebook</figcaption>
```
Comment (ttaannjjaa) [inline index.html:66]: please see comment above and also for line 74

File: .vscode/extensions.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "ms-vscode.live-server"
+    ]
+}
```
Comment (ttaannjjaa) [inline .vscode/extensions.json:5]: sorry to say this... i cannot review this piece of code

**Review by ttaannjjaa — CHANGES_REQUESTED** [review]
Cool things added :-))

**Pull request merged**